### ISSUE TITLE
Divide and conquer mid-year EC update

### ISSUE THREAD

**Issue by brownsarahm** [carpentries/trainers#89]
## Summary
Divide and conquer approach to the mid-year EC update. 

## Background

We owe the EC an [update per the policy](https://docs.carpentries.org/topic_folders/governance/committee-policy.html)
From Paula Andrea
> No special requirements, a one page summary of actions and plans is all we need.

## Implementation

discuss the needed sections in the meeting
preliminary sections:

- summarize proposals
- updates on progress

Do some planning in meeting to produce outline of plans to put in the report. 

### Pros
- we divide the labor

### Cons
- requires more collaboration

## Possible Alternatives

- one person does the report alone

**Comment by brownsarahm** [comment]
possible resource to pull toether report from the repository

**Comment by ACharbonneau** [comment]
Checkout Process Governance

We are in the process of outlining a process for making changes to Trainer Checkout. This document would specify what kinds of changes can be made by which groups, and when decisions require a vote by the Trainer Community. Currently we are identifying stakeholders for various parts of the checkout process, and working to determine who would own the Checkout Governance. 

Leadership Setup

We used the first sessions of the Leadership meetings to outline how this new Leadership group should function. We adopted a variation on Marthas Rules for running our meetings, and developed a procedure for taking and sharing minutes that prioritizes transparency and openness. We elected to do most of our work in GitHub, and have set up a number of templates for issues and pull requests to improve ease of use and to encourage discussion and feedback in this medium.
